Packs/DuoAdminApi/Integrations/DuoEventCollector/DuoEventCollector.py

```diff
@@ -62,7 +69,6 @@ def call(self, request_order: list) -> tuple:
                 elif request_order[0] == LogType.ADMINISTRATION:
                     events = self.handle_administration_logs()
 
-                events = parse_events(events)
```

Comment:
Yes, these are the 3 types we support.
I didn't remove this line, I just relocated it, I moved it to line 321 in the function `_iter_events`. I did it because I wanted this function to run only on the events in the time window and not on all the events that returns from the API call.